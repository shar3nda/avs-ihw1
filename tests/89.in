10
10 10 10 0 0 5 2 6 1 6