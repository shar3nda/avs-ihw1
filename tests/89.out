0 0 5 2 6 1 6